promapp/forms.py

```
@@ -8,6 +8,37 @@
 from parler.forms import TranslatedField
 from django.utils.translation import get_language
 from django.utils.safestring import mark_safe
+from django.core.exceptions import ValidationError
+import magic
+import mimetypes
```

Ruff (0.12.2)

`mimetypes` imported but unused

Remove unused import: `mimetypes`

(F401)